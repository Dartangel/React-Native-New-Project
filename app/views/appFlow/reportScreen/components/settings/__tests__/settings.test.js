import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Settings from '../settings';

describe('Settings', () => {
    test('Settings snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<Settings />);
        expect(snapshot).toMatchSnapshot();
    });
});