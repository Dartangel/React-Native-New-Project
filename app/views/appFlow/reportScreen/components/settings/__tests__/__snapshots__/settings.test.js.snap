// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings Settings snapshot 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "grey",
      "borderBottomWidth": 2,
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
        "padding": 5,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 0.6,
          "justifyContent": "center",
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontSize": 20,
          }
        }
      />
    </View>
  </View>
  <KeyboardAvoidingView
    behavior="padding"
    enabled={true}
    keyboardShouldPersistTaps="always"
    keyboardVerticalOffset="-250"
  >
    <Component
      animationType="fade"
      hardwareAccelerated={false}
      onRequestClose={[Function]}
      transparent={true}
      visible={false}
    >
      <Weight
        onClick={[Function]}
      />
    </Component>
  </KeyboardAvoidingView>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 0.5,
        "justifyContent": "center",
      }
    }
  >
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
      style={
        Object {
          "color": "#08DBA5",
          "fontSize": 15,
        }
      }
      visible={true}
    >
      <Text
        style={
          Object {
            "color": "#08DBA5",
            "fontSize": 15,
          }
        }
      />
    </TouchableOpacity>
  </View>
</View>
`;
